bin/quantile_normalize.sh

```diff
@@ -44,7 +44,7 @@ trap 'rm -f ${tmp_files}.*' INT TERM EXIT
 
 tmp_config_tsv=$tmp_files.config.tsv
 #"$scriptDir/get_assay_groups.pl" "$path_to_experiment_config" > "$tmp_config_tsv"
-"$scriptDir/get_assay_groups.pl" "$path_to_experiment_config" > "$tmp_config_tsv"
+"$scriptDir/atlas-bash-util/get_assay_groups.pl" "$path_to_experiment_config" > "$tmp_config_tsv"
```

Comment:
Are these identical? I remember having to add some changes to the version I brought to this repo. Or maybe it was another function.

Reply:
In this case, they are identical.